modules/op-running-unpriveleged-builds-in-a-container.adoc

```diff
@@ -0,0 +1,51 @@
+[id='op-running-unpriveleged-builds-in-a-container._{context}']
+= Running unprivileged builds in a container
+
+You can run builds as a non-root user inside a container. To run as a non-root user inside the container, your container must first add the `subuid` and `subgid` with the latest shadow-utils.
+
+The public image based on Red Hat UBI has everything already set up for us in its https://catalog.redhat.com/software/containers/ubi8/buildah/602686f7b16b1eb2e30807ee?container-tabs=dockerfile[Containerfile]. As an extra step, you need to “force” running the container as the “build” user since if we leave it to the default, it will be “randomed” and would not have the setup needed to run as non-root.
+
+You can directly edit or export to a specific namespace the buildah ClusterTask as described in the previous section. You can add this `securityContext`` to each build and push steps.
+
+[source,yaml,subs="attributes+"]
+----
+securityContext:
+    runAsUser: 1000
+----
+
+Because you are building as a local user, buildah saves its image cache to `/home/build/.local/cache`. Thus, instead of mounting the containers located in the `/var/lib/` directory, change the `mountPath` for the build to your local user container directory by changing the value for `mountPath` as follows:
+
+[source,yaml,subs="attributes+"]
+----
+volumeMounts:
+    - name: varlibcontainers
+      mountPath: /home/build/.local/share/containers
+----
+
+You can test it using https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md#configmap[ConfigMap workspace] along with a sample DockerFile and a sample taskrun.
```

Comment:
- As mentioned earlier, we cannot include links to GitHub without getting special permission. Instead, I strongly recommend creating a doc topic that covers this same content and adding it to this PR. 
- Also, please do not mark unresolved issues like this one as resolved. If there's some difficulty resolving the issue, I recommend having a conversation with the reviewer.